packages/components/src/Menu/Menu.tsx

```diff
@@ -232,7 +274,13 @@ export function MenuLegacy({
               <div
                 ref={refs.setFloating}
                 className={styles.floatingContainer}
-                {...getFloatingProps()}
+                {...getFloatingProps({
+                  onKeyDown: event => {
```

Comment:
debatable if this is the right way for tabs to behave when it's open. I've seen a few different ways this works; however RAC does this essentially. when the Menu is open, tabs do precisely nothing, so for parity let's just match until a compelling argument to do something else presents itself.